package.json

```diff
@@ -8,17 +8,24 @@
   },
   "scripts": {
     "lit": "rm -rf guide/src/ && paluh-litps --input guide/ --output guide/src/",
-    "test": "npm run-script lit && spago test",
-    "build": "npm run-script lit && spago build"
+    "build": "npm run-script lit && spago build",
+    "test:guide:simpleE2E": "npm run-script lit && spago test --main Test.Guide.SimpleE2E",
+    "test:guide:custom": "npm run-script lit && spago test --main Test.Guide.Custom",
+    "test:guide": "npm-run-all --sequential test:guide:*",
+    "test:main": "spago test",
+    "test": "npm-run-all --sequential test:*"
```

Comment:
separate tests that use pg backend because they may cause conflicts related to transaction.

Reply:
I often got error when running test before
```
| Error: (IntegrityError { code: "23505", column: "", constraint: "pg_type_typname_nsp_index", dataType: "", detail: "Key (typname, typnamespace)=(people, 2200) already exists.", file: "nbtinsert.c", hint: "", internalPosition: "", internalQuery: "", line: "570", message: "duplicate key value violates unique constraint \"pg_type_typname_nsp_index\"", position: "", routine: "_bt_check_unique", schema: "pg_catalog", severity: "ERROR", table: "pg_type", where_: "" })
```

Reply:
There was a transaction overlapping when the error occured.

↓ postgres logs
```
db_1  | 2021-02-05 05:30:01.581 UTC [81] LOG:  connection received: host=192.168.176.1 port=44946
db_1  | 2021-02-05 05:30:01.581 UTC [82] LOG:  connection received: host=192.168.176.1 port=44948
db_1  | 2021-02-05 05:30:01.581 UTC [83] LOG:  connection received: host=192.168.176.1 port=44950
db_1  | 2021-02-05 05:30:01.585 UTC [82] LOG:  connection authorized: user=init database=purspg
db_1  | 2021-02-05 05:30:01.585 UTC [81] LOG:  connection authorized: user=init database=purspg
db_1  | 2021-02-05 05:30:01.586 UTC [83] LOG:  connection authorized: user=init database=purspg
db_1  | 2021-02-05 05:30:01.589 UTC [81] LOG:  statement: BEGIN TRANSACTION
db_1  | 2021-02-05 05:30:01.590 UTC [81] LOG:  statement: 
db_1  | 	  DROP TABLE IF EXISTS people;
db_1  | 	  CREATE TABLE people (
db_1  | 	    id INTEGER PRIMARY KEY,
db_1  | 	    name TEXT NOT NULL,
db_1  | 	    age INTEGER
db_1  | 	  );
db_1  | 2021-02-05 05:30:01.592 UTC [82] LOG:  statement: BEGIN TRANSACTION
db_1  | 2021-02-05 05:30:01.592 UTC [83] LOG:  statement: 
db_1  | 	        DROP TABLE IF EXISTS people;
db_1  | 	        CREATE TABLE people (
db_1  | 	          id INTEGER PRIMARY KEY,
db_1  | 	          name TEXT NOT NULL,
db_1  | 	          age INTEGER
db_1  | 	        );
db_1  | 	
db_1  | 	        DO $$
db_1  | 	        BEGIN
db_1  | 	          IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'account_type') THEN
db_1  | 	            CREATE TYPE ACCOUNT_TYPE as ENUM (
db_1  | 	              'business',
db_1  | 	              'personal'
db_1  | 	            );
db_1  | 	          END IF;
db_1  | 	        END$$;
```

Reply:
It might have been best if it could be split in the code, but it was difficult so I split them using npm-run-all.
@Kamirus How do you think the solution is good?